plugins/Toolbox/resources/qml/ToolboxInstalledTile.qml

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Toolbox is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import UM 1.1 as UM
+
+Item
+{
+    property bool canUpdate: false
+    property bool isEnabled: true
+    height: UM.Theme.getSize("toolbox_installed_tile").height
+    anchors
+    {
+        left: parent.left
+        right: parent.right
+    }
+    Rectangle
+    {
+        color: UM.Theme.getColor("lining")
+        width: parent.width
+        height: UM.Theme.getSize("default_lining").height
+        anchors.bottom: parent.bottom
+    }
+    Column
+    {
+        id: pluginInfo
+        property var color: isEnabled ? UM.Theme.getColor("text") : UM.Theme.getColor("lining")
+        height: parent.height
+        anchors
+        {
+            left: parent.left
+            top: parent.top
+            right: authorInfo.left
+            topMargin: UM.Theme.getSize("default_margin").height
+            rightMargin: UM.Theme.getSize("default_margin").width
+        }
+        Label
+        {
+            text: model.name
+            width: parent.width
+            height: UM.Theme.getSize("toolbox_property_label").height
+            wrapMode: Text.WordWrap
+            verticalAlignment: Text.AlignVCenter
+            font: UM.Theme.getFont("default_bold")
+            color: pluginInfo.color
+        }
+        Text
+        {
+            text: model.description
+            width: parent.width
+            height: UM.Theme.getSize("toolbox_property_label").height
+            clip: true
+            wrapMode: Text.WordWrap
+            color: pluginInfo.color
+            elide: Text.ElideRight
+        }
+    }
+    Column
+    {
+        id: authorInfo
+        height: parent.height
+        width: Math.floor(UM.Theme.getSize("toolbox_action_button").width * 1.25)
+        anchors
+        {
+            top: parent.top
+            topMargin: UM.Theme.getSize("default_margin").height
+            right: pluginActions.left
+            rightMargin: UM.Theme.getSize("default_margin").width
+        }
+        Label
+        {
+            text:
+            {
+                if (model.author_email)
+                {
+                    return "<a href=\"mailto:"+model.author_email+"?Subject=Cura: "+model.name+"\">"+model.author_name+"</a>"
```

Comment:
We are reviewing these e-mails and names to test for code injection, right? :stuck_out_tongue:

Reply:
Yes. Right now it's manually updated but this is eventually a job for whatever interface we build for this.